lua/plenary/iterators.lua

```diff
@@ -107,7 +107,7 @@ local rawiter = function(obj, param, state)
       end
     end
 
-    if vim.tbl_islist(obj) then
+    if vim.islist(obj) then
```

Comment:
No, this will break all old nvim version.

```suggestion
    local islist = vim.islist or vim.tbl_islist
    if vim.islist(obj) then
```

and also for other files.